### ISSUE TITLE
About the cost

### ISSUE THREAD

**Issue by wuhaizhifeng** [divamgupta/ladder_network_keras#3]
I found the code about the cost , you just use the decoder cost(d-cost), however you forgot encoder cost (c-cost)? Right?

**Comment by divamgupta** [comment]
We compile the model with categorical cross entropy, hence that loss is added by keras while compiling.
